lib/booking_bug/api.rb

```diff
@@ -0,0 +1,37 @@
+class BookingBug
+  class UnableToAuthenticate < StandardError; end
+
+  class Api
```

Comment:
Maybe this should be `AdminAPI`?

I'm not sure there is any value in abstracting the fact that this is class is an adapter for the admin API, not all Booking Bug APIs.